### DISCUSSION TITLE
Prepare for new models in roadmap: rnnt

### DISCUSSION THREAD

**Discussion by mrasquinha-g** [mlcommons/inference#1319] [General]
As discussed in the meeting on 1/24, we envision having to retire benchmarks in upcoming submissions. This discussion is to identify potential candidates. Please share your feedback on **rnnt** being a potential candidate.

**Comment by psyhtest** [comment]
While Speech Recognition is useful in general, the current reference implementation is not quantization friendly, in our (admittedly limited) experience.

**Comment by arjunsuresh** [comment]
@mrasquinha-g Instead of having different discussions for individual models can we have a single discussion for all the models in it?
